### DISCUSSION TITLE
Force check for emails?

### DISCUSSION THREAD

**Discussion by RMcNeely** [elementary/mail#773] [Q&A] [answered]
Since the rewrite to Flatpak I've noticed that Mail sometimes doesn't seem to always check for emails on a consistent basis (nothing specific that I can point to as an issue and I'm not sure that I haven't configured something wrong). However I do notice that whenever I open and close the app that it seems to check for mail. So I end up either open and closing mail frequently or I just leave my GMail tab open. Is there a way to force mail to check for messages?

**Comment by marbetschar** [comment, marked as answer]
@RMcNeely I _think_ Mail not fetching new messages is a side effect of the background process crashing, which ultimately _should_ be fixed once https://github.com/elementary/mail/pull/770 is merged and released.

Till then: You can force Mail to fetch new messages for the currently selected folder by clicking on the refresh icon in the conversation list tab bar at the bottom.

**Reply by RMcNeely** [reply]
Great to hear that there may be a fix in the works. IApparently I was somehow on a daily build from 6.2 (probably from testing a previous fix?) So now that I'm updated to v6.4.0  at least I have the refresh icon.  Not having that is what prompted the question originally.
